server/src/main/java/com/objectcomputing/checkins/services/frozen_template/FrozenTemplate.java

```diff
@@ -0,0 +1,133 @@
+package com.objectcomputing.checkins.services.frozen_template;
+
+
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotBlank;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "frozen_templates")
+public class FrozenTemplate {
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "unique id of the feedback template ", required = true)
+    private UUID id;
+
+    @Column(name = "title")
+    @NotBlank
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "title of feedback template", required = true)
+    private String title;
+
+    @Column(name = "description")
+    @Nullable
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "description of feedback template", required = false)
+    private String description;
+
+    @Column(name = "createdBy")
```

Comment:
I know it's been brought up elsewhere but in case it's unnoticed here, camel-case column names aren't working for postgre following the micronaut update.

Reply:
I'll also mention that on the refactor in #1167, the `createdBy` column was renamed to `creatorId`. We could do the same thing here for consistency.

Reply:
Column names changed to snake case!!